src/main/resources/assets/opencomputers/doc/de_DE/block/accessPoint.md

```diff
@@ -0,0 +1,11 @@
+# Access Point
+
+![AAA](oredict:oc:accessPoint)
+
+Der Access Point ist die drahtlose Variante des [Switches](switch.md). Dieser kann benutzt werden um Subnetzwerke zu separieren, so dass [Komponente](../general/computer.md) nicht in anderen Netzwerken erreichbar sind. Dennoch ist es möglich Nachrichten in andere Netzwerke zu schicken.
+
+Als Erweiterung dient dieser Block als Verstärker. Er leitet kabelgebundene Nachrichten als solche weiter, beziehungsweise werden Nachrichten, die über eine Drahtlos-Netzwerkkarte gesendet werden, als Funknachricht oder kabelgebundene Nachricht weiter geleitet.
```

Comment:
`Zusätzlich dient dieser Block als Verstärker.`
`kabelgebundene Nachricht weitergeleitet.`
... Do not translate word-by-word. Please.